src/components/AnimatedStep/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React, {useMemo} from 'react';
 import type {StyleProp, ViewStyle} from 'react-native';
+// eslint-disable-next-line no-restricted-imports -- will be removed in the future PR
 import * as Animatable from 'react-native-animatable';
 import useThemeStyles from '@hooks/useThemeStyles';
-import useNativeDriver from '@libs/useNativeDriver';
```

Comment:
Initially I wanted to migrate animatable to reanimated in one PR but changes from AnimatedStep component become too big to migrate everything in one PR. [Here](https://github.com/Expensify/App/pull/54404/files) is the second PR that will be ready to review after this one is merged.